tests/core/pyspec/eth2spec/test/eip7594/unittests/test_config_invariants.py

```diff
@@ -0,0 +1,20 @@
+from eth2spec.test.context import (
+    spec_test,
+    single_phase,
+    with_eip7594_and_later,
+)
+
+
+@with_eip7594_and_later
+@spec_test
+@single_phase
+def test_invariants(spec):
+    assert spec.FIELD_ELEMENTS_PER_BLOB % spec.FIELD_ELEMENTS_PER_CELL == 0
+    assert spec.FIELD_ELEMENTS_PER_BLOB * 2 % spec.config.NUMBER_OF_COLUMNS == 0
```

Comment:
```suggestion
    assert spec.FIELD_ELEMENTS_PER_EXT_BLOB % spec.config.NUMBER_OF_COLUMNS == 0
```

Low priority. Feel free to merge without.